contrib/systemd/gitea.service

```diff
@@ -53,7 +53,7 @@ WorkingDirectory=/var/lib/gitea/
 # If using unix socket: Tells Systemd to create /run/gitea folder to home gitea.sock
 # Manual cration would vanish after reboot.
 #RuntimeDirectory=gitea
-ExecStart=/usr/local/bin/gitea web -c /etc/gitea/app.ini
+ExecStart=/usr/local/bin/gitea web --config /etc/gitea/conf/app.ini
```

Comment:
I believe docs still recommend having config at `/etc/gitea/app.ini`

Reply:
So there are two `app.ini` files?

Mine was installed under `conf`:
```bash
$ tree -augp
.
`-- [drwxr-xr-x gitea    gitea   ]  conf/
    `-- [-rw-r----- gitea    gitea   ]  app.ini

1 directory, 1 file
```

Reply:
@keks24 there is only one app.ini and default is /etc/gitea/app.ini

if it's not on your system something went wrong

Reply:
@6543 
I installed it according to the documentation and the `conf` directory was generated when I executed `gitea web` as `root`:
```bash
$ cd /usr/local/sbin
$ wget "https://dl.gitea.io/gitea/1.9.5/gitea-1.9.5-linux-amd64" -O gitea
$ chmod 755 gitea
$ mkdir -p /var/lib/gitea/{custom,data,log}
$ chown -R gitea:gitea /var/lib/gitea
$ mkdir /etc/gitea
$ chown root:gitea /etc/gitea
$ chmod 770 /etc/gitea
$ export GITEA_WORK_DIR="/var/lib/gitea" export GITEA_CUSTOM="/etc/gitea" gitea web
$ tree -ugp /etc/gitea
/etc/gitea
`-- [drwxr-xr-x root     root    ]  conf/
    `-- [-rw-r--r-- root     root    ]  app.ini

1 directory, 1 file
```
After that I adapted the permissions and created the systemd service unit:
```bash
$ chown -R gitea:gitea /etc/gitea/conf
$ chmod 750 /etc/gitea
$ chmod 640 /etc/gitea/conf/app.ini
[...]
$ systemctl enable gitea.servce --now
```
Just tested it and it works fine. :)
![Screenshot_20191103_094055](https://user-images.githubusercontent.com/13721064/68082541-4fa1f980-fe1e-11e9-810f-d2d0b5570198.png)

Reply:
According to your above history of commands, you ran without the `-c` flag, so your Gitea used the <custom directory/conf> as a backup.  
This works, however it isn't what the docs outline.

Reply:
OK, now I understand.
I did not know that the configuration file will be generated in that specific location when running the program with `-c <path_to_file>` parameter for the first time. I assumed that it would run into an error because the file was absent. Other programs behave like that.
I revert it. :)

Note to myself: Do not tinker with things and create pull request after 3am in the morning because concentration converges to zero. :D